lib/src/steering/behaviors/seek.dart

```diff
@@ -23,12 +23,12 @@ abstract class Seek extends Behavior {
   }
 
   Seek._(Steerable owner, Vector2 point)
```

Comment:
I'm not familiar with this syntax, what does the `_` mean, is it that the constructor is private? Also can't point here just be `this.target`?

Reply:
Yes, it's a private constructor -- to be used by the derived classes.